.cproject

```diff
@@ -237,4 +238,5 @@
 		</scannerConfigBuildInfo>
 	</storageModule>
 	<storageModule moduleId="org.eclipse.cdt.make.core.buildtargets"/>
+	<storageModule moduleId="refreshScope"/>
```

Comment:
I think our .cproject files will always be different, perhaps is better to ignore them and add a "settings setup" section in the ReadMe. (Add Lib to the include path, untick exclude from build, add support for floating point in printf, etc).

Reply:
Don't know. Perhaps worth researching what .cproject file does.

Reply:
.project and .cproject are the IDE configuration files. .cproject is specific to the CDT (C/C++ plugin) and according to t[his link](https://www.eclipse.org/forums/index.php/t/174633/) 
> This defines things that are specific to C/C++ projects such as include
paths, library paths, and build commands.